jsonrpc/debug_test.go

```diff
@@ -0,0 +1,235 @@
+package jsonrpc_test
```

Comment:
we should have this test on `e2e` tests

Reply:
Issue already created yep: https://github.com/0xPolygonHermez/zkevm-node/issues/1343 plus I supplied code that can be used in a previous comment.